### PR TITLE
Fair Squares W3F grant proposal

### PR DESCRIPTION
**Pull request by ilhanu**
Think we are ready to soon to create a PR in the w3f repo, we just need proof-read the grant proposal and review the parts we write. 

Parts that are missing still
- Visualisation for **milestone 5** 
-  ETH address for grant payments

@ndkazu when you have time can you please re-check the state-diagram and make it coherent with all the naming in the proposal. 

@letodunc added all the sequence diagrams as links in text and please also proof-read and review it.

### CODE REVIEW

**Review by ndkazu — CHANGES_REQUESTED** [review]
Corrected a few type misses, and submitted the updated version of the development state diagram

File: applications/fair_squares.md [hunk 1]
```diff
@@ -0,0 +1,537 @@
...
+    }
+
+    class Tenant_struct{
+      +AccountId AccountId
+      +Balance Rent
+      Pay_rent()
+    }
+    
+```
+
```
[the remark this answers is not in the document]
Comment (ilhanu) [reply applications/fair_squares.md:201]: thanks for the update!

**Review by ilhanu — COMMENTED** [review]
Thanks for the review @ndkazu added all the changes

**Comment by ilhanu** [comment]
Added review Mahir

File: applications/fair_squares.md [hunk 2]
```diff
@@ -0,0 +1,565 @@
...
+
+- Integration into Substrate / Polkadot / Kusama.
+
+The Kusama and Polkadot ecosystem allow us to use the required building blocks through the relay-chain. The substrate framework allows us to configure runtime to the custom needs and bring out a protocol that works like clockwork to onboard assets and rent them out. The already open-sourced frame pallets and other teams building blocks are helping us a lot in achieving the above.
+
+For our social mission to be succesful we require several actions on-chain, such as voting, selection of funds, governing and reccuring payments. We are looking to build a feeless parachain while getting the security from the relay-chain. We believe that the parachain functiunality collaboration is crucial to our mission by using multi-chain services for DIDs, stablecoins, storage protocols within the substrate web3 ecosystem.
```
Comment (letodunc) [inline applications/fair_squares.md:27]: For our mission to be successful

Comment (letodunc) [inline applications/fair_squares.md:27]: We believe that the parachain functionality

File: applications/fair_squares.md [hunk 3]
```diff
@@ -0,0 +1,565 @@
...
+| 2. | **pallet-voting**| We will customizethe known frame pallets such as `democracy` and `collective` pallet and devise to fit the voting structures that are required for FS. The **housing-council** will be the first line of defence and it's structure and selection is very similar to the council as we know from kusama/polkadot. The housing council will be able to asses an asset and approve, reject, request more info. Creating custom calls for the housing council specific to asset information is crucial to determine the state of an asset onboarding. When approved the asset can continue to the investor voting. The **investor-voting** is randomly selected by their role. We can set a minimum of X investors to vote. They will vote from tehir POV whether the asset presented to them is worthy of investing. The investors don't know with certainty that their capital will be allocated on the house their voting. It's being voted from the POV that the housing fund will be used. See also the sequence diagram based for [investor-voting](https://raw.githubusercontent.com/Fair-Squares/fs-docs/dev/docs/diagrams/backend_structure_temlplate_refactor_branch/sequence_diagram_vote_proposal.svg)|
```
Comment (letodunc) [inline applications/fair_squares.md:388]: We will customize the known frame pallet

**Review by letodunc — CHANGES_REQUESTED** [review]

File: applications/fair_squares.md (hunk 2 above)
Comment (ilhanu) [reply applications/fair_squares.md:27]: I won't change this, so make the social mission awareness.

**Comment by ilhanu** [comment]
changed comments @letodunc

**Pull request merged**